alacritty/src/renderer/mod.rs

```diff
@@ -131,6 +135,16 @@ impl Renderer {
             (text_renderer, rect_renderer)
         };
 
+        // Enable debug logging for OpenGL as well.
+        if log::max_level() >= log::LevelFilter::Debug && GlExtensions::contains("GL_KHR_debug") {
+            debug!("Enabled debug logging for renderer");
```

Comment:
Why is this necessary? To see if `GL_KHR_debug` is supported?

Also this should be "debug logging for OpenGL", not renderer.

Reply:
Yeah, I want to check if extension is here.